src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -62,54 +65,72 @@
     public int tumorStandardDeviationsThreshold = 2;
 
     /**
-     * Minimum allele fraction of variant reads in normal for a pileup to be considered inactive
+     * Minimum fraction of variant reads in normal for a pileup to be considered inactive. Applies to normal data in a tumor with matched normal analysis.
+     * For value of 0.1, at least one tenth of pileup must be variant for tool to consider it a germline variant site and therefore not a region of interest
+     * in the somatic analysis.
      */
-    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum number of reads in pileup to be considered active region.")
+    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum fraction of variant reads in normal pileup to be considered germline.")
     public double minNormalVariantFraction = 0.1;
 
     /**
-     * Only variants with tumor LODs exceeding this threshold can pass filtering.
+     * Only variants with tumor LODs exceeding this threshold will be written to the VCF, regardless of filter status.
+     * Set to less than or equal to tumor_lod. Increase argument value to reduce false positives in the callset.
+     * Default setting of 3 is permissive and will emit some amount of false-positives that
+     * filtering should filter and that allows for downstream training outside of the current workflow.
      */
-    @Argument(fullName = "tumor_lod_to_emit", optional = true, doc = "LOD threshold for emit tumor variant")
+    @Argument(fullName = "tumor_lod_to_emit", optional = true, doc = "LOD threshold to emit tumor variant to results.")
     public double emissionLodThreshold = 3.0;
 
     /**
      * This is a measure of the minimum evidence to support that a variant observed in the tumor is not also present in the normal.
+     * Applies to normal data in a tumor with matched normal analysis. The default has been tuned for diploid somatic analyses.
+     * It is unlikely such analyses will require changing the default value. Increasing the parameter may increase the sensitivity of somatic calling,
+     * but may also increase calling false positive, i.e. germline, variants.
      */
-    @Argument(fullName = "normal_lod", optional = true, doc = "LOD threshold for calling normal non-germline")
+    @Argument(fullName = "normal_lod", optional = true, doc = "LOD threshold for calling normal variant non-germline.")
     public double NORMAL_LOD_THRESHOLD = 2.2;
 
     /**
-     * This argument is used for the M1-style strand bias filter
+     * @deprecated This argument is obsolete as of June 2017. It was previously used for the M1-style strand bias filter.
      */
-    @Argument(fullName="power_constant_qscore", doc="Phred scale quality score constant to use in power calculations", optional = true)
+    @Deprecated
+    @Argument(fullName="power_constant_qscore", doc="Phred scale quality score constant to use in power calculations.", optional = true)
```

Comment:
Instead of deprecating, you may delete this argument along with the line `final double errorProbability = QualityUtils.qualToErrorProb(MTAC.POWER_CONSTANT_QSCORE);` in `SomaticGenotypingEngine`.